src/components/scenes/RequestScene.tsx

```diff
@@ -131,17 +129,13 @@ export class RequestSceneComponent extends React.Component<Props, State> {
 
   async generateEncodedUri() {
     const { wallet, currencyCode } = this.props
-    let legacyAddress = ''
     let publicAddress = ''
     if (wallet != null) {
       const receiveAddress = await wallet.getReceiveAddress()
-      // @ts-expect-error
-      legacyAddress = receiveAddress.legacyAddress
       publicAddress = receiveAddress.publicAddress
     }
     this.setState({
-      publicAddress,
-      legacyAddress
+      publicAddress
```

Comment:
I'm ok with this cleanup but note that we will have to add some type of state to track legacy vs public, vs segwit address that the user is looking at. good for now